pom.xml

```diff
@@ -191,6 +228,11 @@
                     <goals>deploy</goals>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>cobertura-maven-plugin</artifactId>
+                <version>${plugins.cobertura.version}</version>
```

Comment:
Also not sure why this plugin is required?

Reply:
If you execute it explicitly (e.g. `mvn clean cobertura:cobertura`) then you don't need to provide it in the POM.  You only need to specify it here if you want it to run on every build.